### DISCUSSION TITLE
thermo loop N dia

### DISCUSSION THREAD

**Discussion by grandrea** [vdemichev/DiaNN#1127] [General]
Hello,

It is common for DIA methods acquired with thermo instruments to use "top N" looping- meaning if the full mass range is covered by 40 windows, but top N is set to for example 20, then in the first duty cycle the instrument acquires the first 20 windows, then in the following duty cycle (after an MS1) it acquires the next 20 windows, and so on. The 2 numbers need not be multiples of each other, which can lead to duty cycles starting on different points of the mass range.

Does DIA-NN parse this correctly from mzML? I see papers published using DIA-NN and such acquisition schemes, but is this supported? or Is it better to have wider windows and set each duty cycle to cover the entire range?

**Comment by vdemichev** [comment]
Hi,

DIA-NN works fine even with some DDA data :)
No, mzIdentML format is not supported.

Best,
Vadim

**Reply by grandrea** [reply]
sorry i meant mzML
